float32 base_pitch
float32 diff_1_left
float32 diff_1_right
float32 diff_2_left
float32 diff_2_right
float32 end_effector
float32 base_yaw

